### PR TITLE
jury marks sans

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

can't really find the kattio snipper, it'd be cool if someone tested the new java code for me

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
damit prettifier

File: solutions/cf-831C.mdx [hunk 1]
```diff
@@ -62,40 +70,75 @@ public class JuryMarks {
...
+			// What all the scores are going to now be given the starting score
+			Set<Integer> points = new TreeSet<>();
 			for (int i = 1; i <= numJury; i++) {
-				points.add(x + marks[i]);
+				points.add(s + changes[i]);
 			}
+			
 			boolean pointsContainsScore = true;
 			for (int p : scores) {
 				flag &= points.contains(p);
```
Comment (TruaShamu) [inline solutions/cf-831C.mdx:99]: Isn't it supposed to be

```suggestion
				pointsContainsScore &= points.contains(p);
```

**Comment by TruaShamu** [comment]
Also, add your name to the authors please?

**Comment by TruaShamu** [comment]
Kattio snippet works, btw here is the Kattio section in the guide if you need: [[LINK]](https://usaco.guide/general/input-output/#method-3---io-template)

**Review by TruaShamu — CHANGES_REQUESTED** [review]
-

**Comment by SansPapyrus683** [comment]
> Also, add your name to the authors please?

don't think it's significant enough to wrarrant that

**Review by TruaShamu — APPROVED** [review]
Alright, it looks good to me!
(And yes, I actually do look at the code in the prs 😡  )

**Review by ryanchou-dev — APPROVED** [review]

**Review by nwatx — CHANGES_REQUESTED** [review]
why are other files being changed

**Comment by nwatx** [comment]
wait nevermind

**Review by nwatx — APPROVED** [review]

**Pull request merged**